frontend/src/lib/components/DataDisplay/GalleryDisplay.svelte

```diff
@@ -108,7 +108,7 @@ $: filteredComponentProps = filteredItems.map((item) => {
 	{/if}
 
 	<Gallery
-		class="grid w-full grid-cols-1 justify-center gap-8 p-4 sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4"
+		class="w-full justify-items-center"
```

Comment:
same here. While the styling isn´t final, some of the other components that use the `GalleryDisplay` depend on it being styled as originally defined and their styling will break if it is changed.

Reply:
Ok, get it.